src/polyfills/stubs/intersection-observer-stub.js

```diff
@@ -268,7 +268,9 @@ export class IntersectionObserverStub {
   upgrade_(Ctor) {
     const inst = new Ctor(this.callback_, this.options_);
     this.inst_ = inst;
-    this.elements_.forEach((e) => inst.observe(e));
+    for (const e of /** @type {!Array<!Element>} */ (this.elements_)) {
```

Comment:
Nit: This is kinda ugly. I would just remove the nullability, and instead truncate arrays when clearing memory.

Reply:
Done via `this.elements_.length = 0` following convention above.